python/python/tests/test_transaction.py

```diff
@@ -282,6 +283,19 @@ async def transaction_test(
     args.append("0-2")
     transaction.xtrim(key11, TrimByMinId(threshold="0-2", exact=True))
     args.append(1)
+
+    min_version = "7.0.0"
+    if not await check_if_server_version_lt(redis_client, min_version):
+        # TODO: change it to pytest fixture after we'll implement a sync client
```

Comment:
Do we have a task for that? If no, please, create.

Reply:
I copied these 3 lines from elsewhere in test_transaction.py, so this comment does not originate from me. I'm not sure if its outdated, it mentions a sync client, which I'm not sure if we're doing? Are you asking me to create a GitHub issue in this repo?

Reply:
I think you can remove this since the comment already exists in `check_if_server_version_lt`